docs/user-guide/configuring.md

```diff
@@ -227,6 +227,8 @@ And in YAML:
 
 These examples allow `var1` to be overwritten in your code, but disallow it for `var2`.
 
+**Note:** Enable the [no-native-reassign](../rules/no-native-reassign.md) rule to prevent overwriting of read-only global variables in your code.
```

Comment:
s/prevent overwriting of/disallow modifications to/

To be consistent with sentence in Rule Details.